src/app/contexts/RequestContext/getStatsDestination/index.js

```diff
@@ -16,6 +16,9 @@ const getStatsDestination = ({ isUK = true, env = 'test', service }) => {
     case 'japanese':
       destination = 'NEWS_LANGUAGES_GNL';
       break;
+    case 'scotland':
+      destination = 'DEFAULT';
```

Comment:
Since this is being updated, we'll need the cypress tests to be updated to include the default bucket.

https://github.com/bbc/simorgh/blob/latest/cypress/support/config/envs.js#L7
`atiAnalyticsDefaultBucket`

ATI cypress test for Canonical:
https://github.com/bbc/simorgh/blob/5cdace08c2180eb22dd8b27b693ecee0c5a1195e/cypress/integration/pages/testsForAllCanonicalPages.js#L25-L62
ATI cypress test for AMP:
https://github.com/bbc/simorgh/blob/5cdace08c2180eb22dd8b27b693ecee0c5a1195e/cypress/integration/pages/testsForAllAMPPages.js#L25-L63

Reply:
Updated :)

Reply:
Great work with updating these Cypress tests.